administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -22,6 +23,61 @@
 $input = $app->input;
 
 $assoc = JLanguageAssociations::isEnabled();
+
+// print_r($input->post);
+
+$component = $input->post->get('component');
+$language = $input->post->get('ref-language');
+
```

Comment:
Anyhow that should go to the associations.php model.

you probably need to overwrite the parent (JModelList) method to get the query `getListQuery` .
https://github.com/jreys/gsoc16_improved-multi-lingual/blob/ba937b48704bbf5cc494f50ac4da406ac4c4908e/administrator/components/com_associations/models/associations.php#L163